AUTHORS.md

```diff
@@ -0,0 +1,22 @@
+<!--
+   - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+   - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+# Authors
+
+- Andy Scherzinger <info@andy-scherzinger.de>
```

Comment:
Is this from another app?
Can't find any copyright that's not me.

Reply:
Nope, I ran

`$ git log --format='%aN <%aE>' | LC_ALL=C.UTF-8 sort -uf`

Reply:
Ah okay